docs/user-manual/en/metrics.md

```diff
@@ -0,0 +1,104 @@
+# Metrics
+
+Apache ActiveMQ Artemis can export metrics to a variety of monitoring systems
+via the [Micrometer](https://micrometer.io/) vendor-neutral application metrics
+facade.
+
+Important runtime metrics have been instrumented via the Micrometer API, and
+all a user needs to do is implement `org.apache.activemq.artemis.core.server.metrics.ActiveMQMetricsPlugin`
+in order to instantiate and configure a `io.micrometer.core.instrument.MeterRegistry`
+implementation. Many implementations of `MeterRegistr` are available from
```

Comment:
sentance seems to end abrubtly i assume you meant to put a link to the micrometer web page with a list of these?